flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -75,4 +84,41 @@ class TableEnvironmentTest {
       "  LogicalTableScan(table=[[default_catalog, default_database, MyTable]])\n"
     assertEquals(expected, actual)
   }
+
+  @Test
+  def testExecuteTwiceUsingSameTableEnv(): Unit = {
+    val settings = EnvironmentSettings.newInstance().useBlinkPlanner().inBatchMode().build()
+    val tEnv = TableEnvironmentImpl.create(settings)
```

Comment:
Use `TableEnvironment` instead of the implementation?

Reply:
use impl class instead of interface class to avoid error: "Static methods in interface require -target:jvm-1.8" in scala